benchmark/install-all.sh

```diff
@@ -19,6 +20,8 @@ tar -xzvf "trivy_${TRIVY_VERSION}_Linux-64bit.tar.gz" trivy
 rm "trivy_${TRIVY_VERSION}_Linux-64bit.tar.gz"
 chmod a+x ./trivy
 
+pip install "cyclonedx-bom==${CYCLONEDXBOM_VERSION}"
```

Comment:
You may want to install that in another separate virtualenv to avoid an observer effect?

Reply:
you think so? I mean, the system under test is a venv created without `--system-site-packages`, so this should not happen.

From a best-practice perspective, you are right. In generally, decoupling a tool into an own venv is probably the best solution here. 
But there was no existing art for my case of installation, so I'd let this decision/task to @sethmlarson. :-D

Reply:
I thought you would be installed in the venv as the code under analysis! ... you are not.